src/elements/kw-share-card/kw-share-card.html

```diff
@@ -145,15 +152,82 @@
     <script>
         Polymer({
             is: 'kw-share-card',
+            behaviors: [
+                Kano.APIClient,
+            ],
             properties: {
-                share: Object
+                liked: {
+                    type: Boolean,
+                    notify: true
```

Comment:
You could set this as a computed property and use `_checkLikes` as the compute method